react/package.json

```diff
@@ -3,13 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@snowplow/browser-tracker": "^3.12.0",
     "@snowplow/browser-plugin-form-tracking": "^3.12.0",
     "@snowplow/browser-plugin-media": "^3.12.0",
     "@snowplow/browser-plugin-youtube-tracking": "^3.12.0",
-    "@snowplow/browser-tracker": "^3.12.0",
+    "@snowplow/browser-plugin-vimeo-tracking": "link:../../snowplow-javascript-tracker/plugins/browser-plugin-vimeo-tracking",
```

Comment:
Reminder for myself, this needs to be changed to the actual plugin once it has been released